libMBIN/Source/NMS/GameComponents/GcSimpleInteractionComponentData.cs

```diff
@@ -5,60 +5,57 @@
 
 namespace libMBIN.NMS.GameComponents
 {
-	[NMS(Size = 0x1C0, Alignment = 0x8, GUID = 0x27B5D767424493C9, NameHash = 0xEAE36C86E1479AF5)]
```

Comment:
again, this Alignment has disappeared...

Reply:
Corrected